mypy_django_plugin/main.py

```diff
@@ -303,11 +304,16 @@ def get_type_analyze_hook(self, fullname: str) -> Optional[Callable[[AnalyzeType
 
     def get_dynamic_class_hook(self, fullname: str) -> Optional[Callable[[DynamicClassDefContext], None]]:
         # Create a new manager class definition when a manager's '.from_queryset' classmethod is called
-        if fullname.endswith("from_queryset"):
+        if fullname.endswith(".from_queryset"):
             class_name, _, _ = fullname.rpartition(".")
```

Comment:
```suggestion
        class_name, _, method_name = fullname.rpartition(".")
        if method_name == "from_queryset":
```
Maybe do this first and just check the method name, since that's needed anyway?